src/ConnectedMode.UnitTests/Migration/MSBuildFileCleanerTests.cs

```diff
@@ -91,7 +91,7 @@ public void Clean_AdditionalFileRefsExist_DifferentProjectKey_SettingsAreNotRemo
         }
 
         [TestMethod]
-        public void Clean_RulesetRefsExist_CorrectProjectKey_SettingsAreRemoved()
```

Comment:
I renamed the existing tests to try to make the differences between the ruleset tests clearer.

We have two sorts of ruleset tests: 
* one the cleans up `Include` elements in `.ruleset` files, and
* the new tests that check `CodeAnalysisRuleSet` properties inside MSBuild project files.